deployment/jobservice/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{ if not .Values.customServiceAccount }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "jobservice.name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "jobservice.labels.all" . | nindent 4 }}
+  {{ if .Values.serviceAccount }}
+  {{ toYaml .Values.serviceAccount }}
+  {{ end }}
+  {{ end }}
```

Comment:
Should the second {{ end }} be at the start of the line?

Reply:
Yep!  Thanks for that.